src/modules/Builder/features/ActivityItemsFlow/ItemFlow/SummaryRow/SummaryRow.tsx

```diff
@@ -34,20 +33,26 @@ export const SummaryRow = ({ name, activityName, 'data-testid': dataTestid }: Su
     [items, activityName, setValue],
   );
 
+  const { question } =
+    ((items ?? []) as Item[]).find(({ id }) => id === getValues(`${name}.itemKey`)) ?? {};
```

Comment:
If you use `ItemFormValues` instead of `Item` we can avoid the `unknown as string` cast below